examples/introduction.ipynb

```diff
@@ -1,8 +1,15 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The widget periodic table"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 10,
```

Comment:
Are you committing on purpose a run notebook?

Reply:
I just commit to save all change. This is not needed to be committed. I will change it back.